charts/eks-anywhere-packages-migrations/templates/_helpers.tpl

```diff
@@ -2,5 +2,9 @@
 Create image name
 */}}
 {{- define "template.image" -}}
```

Comment:
If the package bundle doesn't have tags, why are we using it here? Should this use the digest first and tag second?

Reply:
It's changed back mainly to keep align with the existing two definitions: https://github.com/search?q=repo%3Aaws%2Feks-anywhere-packages+define+%22template.image%22&type=code